src/constants/ValidateSubmitTooltipTitle.js

```diff
@@ -0,0 +1,59 @@
+import { calculationTypeEnums } from "constants/enums";
+
+const validateSubmitTooltip = (
```

Comment:
Could this be move under right under the UserInputForm component? Seeing as this renders stuff conditionally, and is only concerned with the submit button in UserInputForm, it shouldn't be under constants where e.g. enums or default values are

Reply:
yes i tried